### ISSUE TITLE
[General Question] Accessibility Insights + WebView2 app

### ISSUE THREAD

**Issue by kiddliu** [microsoft/accessibility-insights-windows#1623]
With the [latest production update](https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.2334.01), the tool reports lots of 'Chromium components should be scanned with a web-based scanner' after scanned a WebView2-based application, and in the `HOW TO FIX` section it recommendeds Accessibility Insights for Web, which is a extension to Chrome/Edge.

Questions:

1. Should I continue to run tests against WebView2 applications with the Windows tool? Should I stay with the last good version?
2. Despite the tons of warnings, are other reported issues still valid ones?
3. Is there any workaround for the warning?

**Comment by DaveTryon** [comment]
@kiddliu, we made this change because Accessibility Insights for Windows can basically say "Something is wrong with this HTML app", but it can't tell you what to do about it. Accessibility Insights for Web understands HTML and can point out the specific problem. If you can render the HTML in a standalone browser where Accessibility Insights for Web is installed, you'll end up with far more actionable results. 

Errors outside of a Chromium control are still valid and should be fixed. Running an old version will work for a short time, but we only support N-1 version, that option will go away when we ship the next release.

**Comment by kiddliu** [comment]
Hey @DaveTryon, WebView-based applications usually have critical dependencies on the injected native objects and it may require a huge effort to make it renderable in the browser. Is it possible for the Windows version to support WebView2 apps natively in the future?

**Comment by DaveTryon** [comment]
@kiddliu, that's an intriguing idea! Realistically, though, it's probably way beyond the scope of what our team is able to take on. I _could_ envision a scenario where an app hooks the loader, then runs an HTML scanner (either axe-core or https://www.npmjs.com/package/accessibility-insights-scan) to capture the results. It probably would require an instrumented build to connect the scanner to the load hook. You're free to file a feature request at https://github.com/microsoft/accessibility-insights-web/issues/new/choose, but our team's backlog is already stretched pretty thin, so there are no guarantees that we'd be able to pick it up.

**Comment by kiddliu** [comment]
Will give the scan CLI a try, it looks quite promising. Thank you, @DaveTryon
